### PR TITLE
Improve execution context logging

### PR DESCRIPTION
**Pull request by grouma**

### CODE REVIEW

File: dwds/lib/src/debugging/execution_context.dart [hunk 1]
```diff
@@ -25,22 +27,26 @@ class RemoteDebuggerExecutionContext extends ExecutionContext {
...
         });
         if (result.result['result']['value'] != null) {
+          _logger.fine('Found valid execution context: $context');
           _id = context;
           break;
         }
       } catch (_) {
         // Errors may be thrown if we attempt to evaluate in a stale a context.
         // Ignore and continue.
+        _logger.fine('Invalid exeuction context: $context');
```
Comment (annagrin) [inline dwds/lib/src/debugging/execution_context.dart:49]: nit: typo in 'execution'

**Review by devoncarew — APPROVED** [review]

File: dwds/lib/src/debugging/execution_context.dart [hunk 2]
```diff
@@ -25,22 +27,26 @@ class RemoteDebuggerExecutionContext extends ExecutionContext {
...
             await _remoteDebugger.sendCommand('Runtime.evaluate', params: {
           'expression': r'window["$dartAppInstanceId"];',
           'contextId': context,
         });
         if (result.result['result']['value'] != null) {
+          _logger.fine('Found valid execution context: $context');
           _id = context;
           break;
         }
       } catch (_) {
```
Comment (annagrin) [inline dwds/lib/src/debugging/execution_context.dart:46]: will it be useful to print the exception message as well?

**Review by annagrin — APPROVED** [review]
LGTM (with comments)

File: dwds/lib/src/debugging/execution_context.dart (hunk 2 above)
Comment (grouma) [reply dwds/lib/src/debugging/execution_context.dart:46]: That should get logged by whatever captures the error.

**Pull request merged**